### ISSUE TITLE
TODO for suggestions module

### ISSUE THREAD

**Issue by mathematicalmichael** [mathematicalmichael/spellfix#17]
not yet implemented:
- [ ] argparse
- [ ] not part of `setup.py`
- [ ] testing

**Comment by mathematicalmichael** [comment]
low priority because it _does_ work. 

arguments:
- threshold for similarity (set at 75% now because 80% wasn't automating shorter words)
- n-gram length

**Comment by mathematicalmichael** [comment]
make sure that the program can run even without the presence of matches. think of this as an "extra feature", however it is generated. 

right now, the package doesn't require `pandas` or anything like that, which is pretty convenient, although inefficient since it's reading files. 

one possible solution that "meets in the middle" is to use `pandas` within `suggest` to create a new dataframe, with each column name being what is currently used as the file name, and the entries for each are below. Then you can save it as a JSON and read that in as an extra data-source once (on startup). this minimizes I/O operations to disk, but does increase the RAM that the program needs.
> TL;DR make dictionary, save and load once, instead of saving and reading lots of files.
